spec/requests/stats_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Stats API", type: :request do
+  describe "/stats" do
```

Comment:
Would you be happy to see if there is a way of testing the response type as a test? E.g. the response was JSON.

One of the requirements of this is the response should be JSON, so it would be nice to see this evidenced in the unit tests.

Reply:
Something like `JSON.parse(response.body)` should do the trick :) you might need to `require 'json'`

Reply:
As far as I can see there's no inbuilt way of testing that a response is JSON. But we could attempt to parse the response and expect it to not raise an exception?

```ruby
    it "is JSON" do
      get request_path
      expect { JSON.parse(response.body) }.to_not raise_exception
    end
```

Reply:
> Something like `JSON.parse(response.body)` should do the trick :) you might need to `require 'json'`

Ah thanks, this comment wasn't showing up when I replied to Alan! 😄 Yes it's the solution I've gone with for the time being.